README.md

```diff
@@ -202,9 +202,16 @@ _Note: the alpha releases on Android are automatically carried by the `alpha-rel
 
 For this step you’ll need to have a proper iOS development certificate on your dev machine that is also installed on your physical device.
 
-```
-react-native run-ios --configuration Release --device 'YOUR_DEVICE_NAME'
-```
+To test the io-app on a real iOS device you must:
+1. Open the project with Xcode and modify the bundle identifier (eg: add ‘.test’ to the existing one)  
+1. Go to the 'Build Settings' tab and in the PROVISIONING_PROFILE section delete the existing ID. Then select 'ios developer' in the debug field of the 'Code Signing Identity'  
+1. In General tab select the 'Automatically Menage Signing' checkbox  
+1. You must have an Apple id developer and select it from the 'Team' drop-down menu  
+1. (Without Xcode) navigate in the io-app project and open the package.json file, in the scripts section add: _"build: ios": "react-native bundle --entry-file = 'index.js' - bundle-output = '. / ios / main.jsbundle' --dev = false --platform = 'ios' "_ 
+1. Open the Terminal and from the root directory project run _npm run build: ios_  
+1. In Xcode navigate in the project, select _'main.jsbundle'_ and enable the checkbox on the right labeled 'ItaliaApp'
+1. Clean and launch the project on the iOS device, if the unit tests fail they can be disabled by going to Product -> Scheme -> Edit Scheme -> Build
```

Comment:
"launch the project" what does it mean? if you can put more detailed informations about this step